Dropdown/Dropdown.js

```diff
@@ -121,6 +138,17 @@ const DropdownBase = kind({
 		open: false
 	},
 
+	handlers: {
```

Comment:
We need to consider `onKeyDown`handler to close when press left or right button to close dropdown?
I am leaving the comment for future discussion